DigitalLearningSolutions.Web/Views/Shared/Components/TextInput/Default.cshtml

```diff
@@ -0,0 +1,21 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common
+@model TextInputViewModel
+
+<div class="nhsuk-form-group @(Model.HasError?"nhsuk-form-group--error":"")">
+  <label class="nhsuk-label" for="@Model.Name">
+    @Model.Label
+  </label>
+  @if (Model.HasError) {
+    <span class="nhsuk-error-message" id="@Model.Name-error">
+      <span class="nhsuk-u-visually-hidden">Error:</span> @Model.ErrorMessage
+    </span>
+  }
+  <input class="nhsuk-input nhsuk-u-width-one-half @(Model.HasError?"nhsuk-input--error":"")" 
```

Comment:
Super minor, but having some spacing around the ? and : in ternary makes it a bit easier to read. Maybe worth running a formatter on the file?